src/main/java/doob/util/SoundManager.java

```diff
@@ -0,0 +1,53 @@
+package doob.util;
+
+import javafx.scene.media.Media;
+import javafx.scene.media.MediaPlayer;
+
+/**
+ * Class to manage the soundeffects in the game.
+ */
+public final class SoundManager {
+	
+	private static MediaPlayer tune;
+	private static MediaPlayer soundEffect;
+	private static double volume;
+	
+	public static final String DIE_EFFECT = "/Sounds/aww.mp3";
+	public static final String POP_EFFECT = "/Sounds/pop.mp3";
+	public static final String LEVEL_COMPLETE = "/Sounds/congratulations.mp3";
+	public static final String GAME_WON = "/Sounds/applaus.mp3";
+	public static final int MAX_VOLUME = 100;
+	
+	private SoundManager() {
+	}
+	
```

Comment:
No than it gives a checkstyle error that a public or default constructor is now allowed.